contrib/localnet/orchestrator/smoketest/runner/bitcoin.go

```diff
@@ -19,9 +21,8 @@ import (
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/rs/zerolog/log"
 	"github.com/zeta-chain/zetacore/common"
-	"github.com/zeta-chain/zetacore/common/bitcoin"
 	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
-	"github.com/zeta-chain/zetacore/zetaclient"
+	zc_btc "github.com/zeta-chain/zetacore/zetaclient/bitcoin"
```

Comment:
Using underscore is not conventional for imports.
We could use something like `zetabitcoin`